File: Editorial/68_Text_Justification/Go/1_Approach.go
func getWords(i int, words []string, maxWidth int) []string {
    currentLine := []string{}
    currLength := 0
    for i < len(words) && currLength+len(words[i]) <= maxWidth {
        currentLine = append(currentLine, words[i])
        currLength += len(words[i]) + 1
        i++
    }
    return currentLine
}

File: Editorial/68_Text_Justification/C/1_Approach.c
struct answer {
    char** arr;
    int size;
};
struct answer getWords(int i, char** words, int wordsSize, int maxWidth) {
    struct answer currentLine;
    currentLine.arr = (char**)malloc(wordsSize * sizeof(char*));
    currentLine.size = 0;
    int currLength = 0;
    while (i < wordsSize && currLength + strlen(words[i]) <= maxWidth) {
        currentLine.arr[currentLine.size++] = words[i];
        currLength += strlen(words[i]) + 1;
        i++;
    }
    return currentLine;
}
